src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -17,15 +17,18 @@
     private final TaskRepository taskRepository;
     private final AdminConfig adminConfig;
 
-    //@Scheduled(fixedDelay = 10000)
     @Scheduled(cron = "0 0 10 * * *")
     private void sendInformationEmail() {
         long size = taskRepository.count();
+        String pluralWord = " tasks.";
+        if (size == 1) {
+            pluralWord = " task.";
+        }
```

Comment:
Ok. chociaz mozna zapisac to krocej
`pluralWord = size == 1 ? " task." : " tasks"`